test/v200/utils/common/component_test_utils.go

```diff
@@ -0,0 +1,172 @@
+package common
+
+import (
+	"fmt"
+	"strconv"
+
+	schema "github.com/devfile/api/v2/pkg/apis/workspaces/v1alpha2"
+)
+
+// componentAdded adds a new component to the test schema data and to the parser data
+func (devfile *TestDevfile) componentAdded(component schema.Component) {
+	LogInfoMessage(fmt.Sprintf("component added Name: %s", component.Name))
+	devfile.SchemaDevFile.Components = append(devfile.SchemaDevFile.Components, component)
+	if devfile.Follower != nil {
+		devfile.Follower.AddComponent(component)
+	}
+}
+
+// componetUpdated updates a component in the parser data
+func (devfile *TestDevfile) componentUpdated(component schema.Component) {
+	LogInfoMessage(fmt.Sprintf("component updated Name: %s", component.Name))
+	if devfile.Follower != nil {
+		devfile.Follower.UpdateComponent(component)
+	}
+}
+
+// addVolume returns volumeMounts in a schema structure based on a specified number of volumes
+func (devfile *TestDevfile) addVolume(numVols int) []schema.VolumeMount {
+	commandVols := make([]schema.VolumeMount, numVols)
+	for i := 0; i < numVols; i++ {
```

Comment:
same as https://github.com/devfile/api/pull/348/files#r579477717

Reply:
Also in this case I am adding entries to a map, not iterating through it, so I don't think range works.